### ISSUE TITLE
Suggestions for speeding up generation?

### ISSUE THREAD

**Issue by jike212** [danielwestendorf/breezy-pdf-lite#151]
This is a great project, super simple and straight-to-the-point, thank you!

I'm testing rendering times and it seems that no matter how fast the machine I run it on, small PDFs tend to take no less than 5 seconds to generate which seems like quite a while. I'm running it in Docker with no environment flags except for the token so debug overhead shouldn't be there.

Any suggestions on how to bring down the time to generate a PDF?

**Comment by danielwestendorf** [comment]
Are you specifying a [completion trigger](https://github.com/danielwestendorf/breezy-pdf-lite#2b-completion-triggers)? The [default trigger](https://github.com/danielwestendorf/breezy-pdf-lite/blob/master/lib/pdf/completion-trigger.js#L21) is a timer of 5 seconds.

**Comment by jike212** [comment]
So I added the following lines to my HTML and tried setting it to `0` but saw in the logs that it wasn't overriding the default `5000`. I then changed it to `1` and it brought the generation time down to about 1.2 seconds which is great.
```
<meta name="breezy-pdf-timer" content="1">
<meta name="breezy-pdf-timeout" content="1">
```
Is there a way to entirely disable any completion triggers or timeouts? I'm just looking for a way to return the PDF as quickly as possible.
